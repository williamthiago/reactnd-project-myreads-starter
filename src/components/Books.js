import React from 'react'

import Book from './Book'

const Books = ({ books, onChangeShelf }) => (
  <ol className="books-grid">
    {books.map(book => (
      <li key={book.id}>
        <Book book={book} onChangeShelf={onChangeShelf} />
      </li>
    ))}
  </ol>
)

export default Books