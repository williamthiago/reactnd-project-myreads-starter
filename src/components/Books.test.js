import React from 'react'
import { shallow } from 'enzyme'

import Books from './Books'

it('Renders Books without books', () => {
  const books = []
  const onChangeShelf = () => {}
  expect(shallow(<Books books={books} onChangeShelf={onChangeShelf} />)).toMatchSnapshot()
})