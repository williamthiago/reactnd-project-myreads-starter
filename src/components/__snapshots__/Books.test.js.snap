// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Books without books 1`] = `
<ol
  className="books-grid"
/>
`;
